Add database folder to .gitignore

yair opened the pull request.

yair committed:
Add database folder to .gitignore

yair committed:
Some refactoring

1. Changed database key serialization from manual methods to using
bincode with the big_endian and fixint_encoding options. This is
bitwise-identical to the original keys, but less verbose.

2. Changed other manual conversions of vectors to fixed-size arrays to
using the clone_from_slice function. This can probably be done less
verbosely as well.

yair committed:
Some refactoring

Changed other manual conversions of vectors to fixed-size arrays to
using the clone_from_slice function. This can probably be done less
verbosely as well, probably using bincode.

ethernomad commented:
Thanks for this - can you break it into 3 pull requests?

database folder to .gitignore
using clone_from_slice
using bincode to encode db keys

yair committed:
Merge branch 'tests' of ssh://github.com/yair/acuity-atomic-swap-offchain into tests

yair committed:
Some more refactoring

Changed database key serialization from manual methods to using
bincode with the big_endian and fixint_encoding options. This is
bitwise-identical to the original keys, but less verbose.

yair commented:
Now there's 5 commits. 8|
